### PR TITLE
[office-js, office-js-preview] Update Visio and OneNote types to match Office CDN

### PR DESCRIPTION
**Pull request by AlexJerabek**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://appsforoffice.microsoft.com/lib/1/hosted/office.debug.js
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/office-js/index.d.ts [hunk 1]
```diff
@@ -93293,159 +93962,192 @@ declare namespace Visio {
...
-         * Queues up a command to load the specified properties of the object. You must call "context.sync()" before reading the properties.
+        readonly text: string;
+        /** Sets multiple properties of an object at the same time. You can pass either a plain object with the appropriate properties, or another API object of the same type.
          *
          * @remarks
          *
-         * In addition to this signature, this method has the following signatures:
+         * This method has the following additional signature:
+         *
+         * `set(properties: Visio.Shape): void`
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js/index.d.ts:94104]: We don't still need to note additional signatures like this, right?

File: types/office-js/index.d.ts [hunk 2]
```diff
@@ -93500,813 +94214,657 @@ declare namespace Visio {
...
-         * Returns the view of the shape. Read-only.
-         *
-         * [Api set:  1.1]
-         */
-        readonly view: Visio.ShapeView;
+    interface MasterInfo {
         /**
          *
-         * Shape's identifier. Read-only.
+         * Localised Display Name of the Master
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js/index.d.ts:94224]: Perhaps should be something like this?
```suggestion
         * Localized display name of the master.
```

File: types/office-js/index.d.ts [hunk 3]
```diff
@@ -93500,813 +94214,657 @@ declare namespace Visio {
...
          *
          * [Api set:  1.1]
          */
-        readonly id: number;
+        name: string;
         /**
          *
-         * Shape's name. Read-only.
+         * Master Name
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js/index.d.ts:94231]: Here and other Visio descriptions missing period at the end.
```suggestion
         * Master name.
```

Comment (AlexJerabek) [reply types/office-js/index.d.ts:94231]: I did a quick pass to add periods and adjust capitalization. TBH, there are so many other issues (some existing, some new), that I think it's better to fix the base file and rerun codegen. I'm pushing this to get parity with the CDN, but prioritizing the actual update will be a different problem.

**Review by ElizabethSamuel-MSFT — APPROVED** [review]

**Pull request merged**